demo/site/src/util/createSitePath.ts

```diff
@@ -0,0 +1,19 @@
+type CreateSitePathOptions = {
+    path: string;
+    scope: {
+        language: string;
+    };
+};
+
+/**
+ * Creates a Site path from the provided options.
+ *
+ * @return {string} The resolved URL: /{scope.language}/path
+ */
+export const createSitePath = ({ path, scope }: CreateSitePathOptions) => {
+    if (path.startsWith("/")) {
+        return `/${scope.language}${path}`;
+    }
+
+    return `/${scope.language}/${path}`;
```

Comment:
shall we automatically add an missing / or fail if the path does not start with a /

Reply:
automatically adding will be most probably wrong - better fail